test/uk/gov/hmrc/personaldetailsvalidation/PersonalDetailsValidationResourceControllerSpec.scala

```diff
@@ -79,7 +79,7 @@ class PersonalDetailsValidationResourceControllerSpec
     forAll(ninoTransformationScenarios) { (scenario, originalNinoValue, finalNinoValue) =>
       s"$scenario" in new Setup {
 
-        val requestPersonalDetails = randomPersonalDetails
+        val requestPersonalDetails : PersonalDetailsWithNino = randomPersonalDetails.asInstanceOf[PersonalDetailsWithNino]
```

Comment:
Why is typecast required?

Reply:
I think we should then change the type of `randomPersonalDetails` from PersonalDetails to `PersonalDetailsWithNino` because that is what is getting created.